### PR TITLE
Add menu item "contact" to navigation bar

### PR DESCRIPTION
**Pull request by theresantonie**
### Short description

The PR adds the menu item "contacts" to the navigation bar for the new feature "contacts information"

### Proposed changes

- Add menu item "contact" 
- As suggested in the meta issue #2955 I used manage_translations for the permissions

### Side effects

- Unblock other issues of the meta issue 

### Resolved issues

Fixes: #2951

__________________________________________________

[Pull Request Review Guidelines](https://digitalfabrik.github.io/integreat-cms/pull-request-review-guide.html)

### CODE REVIEW

File: integreat_cms/cms/templates/_base.html [hunk 1]
```diff
@@ -187,6 +187,14 @@
                         </a>
                     </div>
                 {% endif %}
+                {% if perms.cms.manage_translations %}
```
Comment (JoeyStk) [inline integreat_cms/cms/templates/_base.html:190]: This is quite unfortunate, but the screenshot is a bit outdated. In the meantime we decided to move the contact below POI/locations. Could you please move this section below POI? Thank you so much & I'm sorry for the confusion!

Comment (theresantonie) [reply integreat_cms/cms/templates/_base.html]: Thank you for the info.
I have updated `_base.html` accordingly.

**Review by JoeyStk — APPROVED** [review]
Thank you very much! Looks great! :)

File: integreat_cms/cms/templates/_base.html [hunk 2]
```diff
@@ -195,6 +195,14 @@
...
```
Comment (david-venhoff) [inline integreat_cms/cms/templates/_base.html:198]:
```suggestion
                {% if perms.cms.view_contact %}
```

**Review by david-venhoff — APPROVED** [review]

**Pull request merged**